dev/PushNotifications/PushNotificationRegistrationToken.h

```diff
@@ -0,0 +1,22 @@
+﻿#pragma once
+#include "Microsoft.Windows.PushNotifications.PushNotificationRegistrationToken.g.h"
+
+namespace winrt::Microsoft::Windows::PushNotifications::implementation
+{
+    struct PushNotificationRegistrationToken : PushNotificationRegistrationTokenT<PushNotificationRegistrationToken>
+    {
+        PushNotificationRegistrationToken(uint64_t const& cookie, winrt::Windows::ApplicationModel::Background::BackgroundTaskRegistration const& taskRegistration);
+        uint64_t Cookie();
```

Comment:
NIT: const

`uint64_t Cookie() const;`

Reply:
It doesn't compile.